install/helm/agones/templates/crds/fleetallocation.yaml

```diff
@@ -25,6 +25,22 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.gameServer.status.state
+    name: State
+    type: string
+  - JSONPath: .status.gameServer.status.address
+    name: Address
+    type: string
+  - JSONPath: .status.gameServer.status.ports[?(@.name=='default')].port
```

Comment:
What happens if the user doesn't have a "default" port - does nothing show?

Reply:
That's correct.

Reply:
I'm wondering if we don't want to show the entire map (probably too wide?) - should we grab the first port in the list, rather than one with the name "default" - just in case? That way we always get a record?

@Kuqd what do you think?

Reply:
Can we require GS to always have a 'default' port?

Reply:
I'm also for either the first one or all of them. Don't have a strong opinions, if it's easier the first one would do for now.

Reply:
How about we start with the first one for now - as that is the most simple, and doesn't break anything for anyone. If we find that people really want all ports later on, we can adjust it, or see if there is a way to make it work in a `o  -wide` scenario.

Also, reorganising the order of ports in the list for output display is far less brittle for users, than forcing people into a naming convention that may not work with their systems.

WDYT?

Reply:
Updated GS and FleetAllocation CRD to ports[0]